packages/chrome/README.md

```diff
@@ -0,0 +1,11 @@
+# Theme UI Devtools
```

Comment:
Sorry haven't had a ton of time to dig in yet, but planning on going through this again soon. I *think* Chrome might need a `manifest.json` file to load the plugin. I had to create one locally at least

Reply:
@jxnblk I’m such an idiot. I’ve moved all the files required for the plugin  into the “build” folder. That way you dont load all of the src or node modules. And you load the chrome plugin from the build folder too. But I’ve realised that folder is in gitignore that’s why it’s not there. It should definitely be there, I’ve made icons too. I’ll have them up in the next few hours.

Reply:
Ha! No worries. I figured it might've been a gitignore issue – looking forward to trying it out!

Reply:
Hey @jxnblk , I've fixed up that gitignore issue. 

So all the files required for the chrome extension are in the `packages/chrome/public` folder. When you add this to chrome just make sure you specify that path and not `packages/chrome`.

Note: You will need to run `yarn dev:editor` and `yarn dev:chrome` to build and watch the bundle.